README.md

```diff
@@ -24,31 +24,68 @@ Create a new temporary project:
 * With a dependency that have a fixed version:
     `$ cargo-temp anyhow==1.0.13`
 
+### Repositories
+
+You can add repositories to you Cargo.toml.
```

Comment:
```suggestion
You can add repositories to your Cargo.toml.
```

Reply:
You can wrap `Cargo.toml` in backticks also

Reply:
Done, thanks!